test/ranges-test.cc

```diff
@@ -90,6 +90,18 @@ TEST(ranges_test, format_tuple) {
       std::tuple<int, float, std::string, char>(42, 1.5f, "this is tuple", 'i');
   EXPECT_EQ(fmt::format("{}", t), "(42, 1.5, \"this is tuple\", 'i')");
   EXPECT_EQ(fmt::format("{}", std::tuple<>()), "()");
+
+  enum class noformatenum{b};
+  struct noformatstruct{};
```

Comment:
Only one unformattable type is needed. Please reuse existing `unformattable` struct (you might need to move it outside of conditional compilation block).

Reply:
Fixed.